docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug: `1311` Fix app bar being visible for logged out users.
```

Comment:
Don't fix now,  I will do in the bump PR.

But remember to keep the changelog style consistent. That would be: "App bar is no longer visible for logged out users".